tests/unit_tests/example_test/Makefile.am

```diff
@@ -0,0 +1,13 @@
+AUTOMAKE_OPTIONS = foreign
+
+check_PROGRAMS = testdriver testdriver2
+
+TESTS = testdriver testdriver2
```

Comment:
Why not `TEST = $(check_PROGRAMS)`?  I'd prefer to only have to maintain a single list :)